### PR TITLE
show object detected file, if object detection in place

### PR DESCRIPTION
**Pull request by pliablepixels**
Allows us to see the objects detected, if ES is in use and notes contains object detection.

Screenshots:

Elements with object detection are hyperlinked:

screen shot 2019-02-10 at 4 25 04 pm

On click:
screen shot 2019-02-10 at 4 25 21 pm

### CODE REVIEW

File: web/skins/classic/views/events.php [hunk 1]
```diff
@@ -274,25 +294,25 @@
 }
 ?>
         <div id="contentButtons">
-          <button type="button" name="viewBtn" value="View" data-on-click-this="viewEvents" disabled="disabled">
+          <button type="button" name="viewBtn" value="View" onclick="viewEvents(this, 'markEids');" disabled="disabled">
```
Comment (mnoorenberghe) [inline web/skins/classic/views/events.php:297]: We are removing inline event listeners and banning them via CSP with script-src. I guess you accidentally reverted these.

Comment (pliablepixels) [reply web/skins/classic/views/events.php:297]: uh oh

Comment (pliablepixels) [reply web/skins/classic/views/events.php:297]: is your change merged to master? maybe that's why I'm not seeing those changes

Comment (pliablepixels) [reply web/skins/classic/views/events.php:297]: Never mind, stupid mistake. Will fix.

**Comment by pliablepixels** [comment]
I'm going to do a new PR, merging this old branch is confusing

**Pull request closed without merging**